lib/api/apiUtils/integrity/validateChecksums.js

```diff
@@ -0,0 +1,27 @@
+const crypto = require('crypto');
+
+const ChecksumError = Object.freeze({
+    MD5Mismatch: 'md5 mismatch',
+    MissingChecksum: 'missing checksum',
```

Comment:
```suggestion
    MD5Mismatch: 'MD5Mismatch',
    MissingChecksum: 'MissingChecksum',
```

I'd prefer the same string for key and value

Reply:
ok